src/js/other-websites/fill_relay_address.js

```diff
@@ -20,6 +20,26 @@ function fillInputWithAlias(emailInput, relayAlias) {
 
 }
 
+function fillInputWithRelayNumber(emailInput, relayNumber) {
+
+  // BUG: Duplicate fillInputWithAlias calls without proper input content
+  // The relayNumber/emailInput arguments check below is a work-around to let the duplicate call(s) fail silently.
+  // To debug, check all instances where fillInputWithRelayNumber() is being called and isolate it.
+  if (!emailInput || !relayNumber) {
+    return false;
+  }
+
+  // Set the value of the target field to the mask number
+  const normalisedNumber = relayNumber.number.startsWith("+1")
```

Comment:
question: I know this will become an issue if we launch outside of the US. Is there a way to dynamically catch the `+1` without hardcoding?

Reply:
Yeah we're going to have a lot of rework to do (not just here, but in the website too) if we expand beyond `+1`, and I unfortunately don't have a good answer to how to dynamically catch it. I think the best we can do when we expand is have a list of all possibile country codes...

That said, in this case it might not be too much of a problem: if the phone number doesn't start with a `+1`, it won't be stripped off, and we'll just insert the phone number including a country code. We're already accepting that this prefill will be imperfect for many possible validations. And since some forms might actually _require_ the country code, it will probably just work in different situations.

Reply:
You called out the higher level answer: expanding outside of `+1` is going to be a lot of work the whole app, not just here, subtly in the add-on.